src/Export/Adapters/ExportItemAdapter.php

```diff
@@ -162,14 +162,16 @@ public function adaptVariant(Item $item, ProductEntity $product): ?Item
                 $item->addOrdernumber($orderNumber);
             }
 
+            foreach ($this->adapterFactory->getAttributeAdapter()->adapt($product) as $attribute) {
+                $item->addMergedAttribute($attribute);
+            }
+
             foreach ($this->adapterFactory->getShopwarePropertiesAdapter()->adapt($product) as $property) {
                 $item->addProperty($property);
             }
 
-            $attributes = $this->adapterFactory->getVariantConfigurationAdapter()
-                ->getOptionAttributes($product);
-
-            foreach ($attributes as $attribute) {
+            foreach ($this->adapterFactory->getVariantConfigurationAdapter()
+                         ->getOptionAttributes($product) as $attribute) {
```

Comment:
If this is reaching the max line length, please keep it in the variable, because this looks very weird 😅 

Or maybe try it like this:
```
foreach(
    ...
) {
}
```